tests/unit/utils/test_metadata_utils.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+
+from amundsen_application.api.utils.metadata_utils import marshall_table_full
+from amundsen_application import create_app
+
+local_app = create_app('amundsen_application.config.TestConfig', 'tests/templates')
+
+
+class MetadataUtilsTest(unittest.TestCase):
```

Comment:
This isn't exactly what I meant but I can modify this after merging and tag you for reference. Ideally, we would have already had a `test_metadata_utils` set up with at least one example to make things easier.